src/pages/basic_data/app_admin/central_import/CentralImportPage.tsx

```diff
@@ -1,17 +1,17 @@
 import { Divider, List, TablePagination, Typography } from '@mui/material';
 import { useTranslation } from 'react-i18next';
-import { useFetch } from '../../../utils/hooks/useFetch';
-import { Registration } from '../../../types/registration.types';
-import { SearchApiPath } from '../../../api/apiPaths';
-import { SearchResponse } from '../../../types/common.types';
-import { ListSkeleton } from '../../../components/ListSkeleton';
+import { useFetch } from '../../../../utils/hooks/useFetch';
+import { Registration } from '../../../../types/registration.types';
+import { SearchApiPath } from '../../../../api/apiPaths';
+import { SearchResponse } from '../../../../types/common.types';
+import { ListSkeleton } from '../../../../components/ListSkeleton';
 import { CentralImportResultItem } from './CentralImportResultItem';
-import { dataTestId } from '../../../utils/dataTestIds';
-import { ROWS_PER_PAGE_OPTIONS } from '../../../utils/constants';
-import { SearchParam } from '../../../utils/searchHelpers';
+import { dataTestId } from '../../../../utils/dataTestIds';
+import { ROWS_PER_PAGE_OPTIONS } from '../../../../utils/constants';
+import { SearchParam } from '../../../../utils/searchHelpers';
 import { useEffect } from 'react';
 import { useHistory } from 'react-router-dom';
-import { stringIncludesMathJax, typesetMathJax } from '../../../utils/mathJaxHelpers';
+import { stringIncludesMathJax, typesetMathJax } from '../../../../utils/mathJaxHelpers';
```

Comment:
denne sluttet plutselig å fungere for meg

Reply:
hm, ser ut som det har skjedd noe der ja... det var nytt for meg